src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/azure-functions/dotnet:3.0 AS base
+WORKDIR /app
+
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:2.2-alpine3.8 AS build
```

Comment:
Are you using 2.2 on purpose here?

Reply:
Yeah, I was searching for the right supported image or so.

Reply:
Removed this, resolving.